.github/workflows/json-check.yml

```
@@ -11,7 +11,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

You might as well just make them use latest; all our other actions do:
```suggestion
    runs-on: ubuntu-latest
```